urbackupserver/www2/src/features/backups/makeBackupsBreadcrumbs.ts

```diff
@@ -0,0 +1,77 @@
+import { BASE_HREF, BreadcrumbItem } from "../../components/Breadcrumbs";
+import { formatDatetime } from "../../utils/format";
+
+export function makeBackupsBreadcrumbs({
+  clientId,
+  clientName,
+  backupId,
+  backuptime,
+  path,
+}: {
+  clientId: number;
+  clientName: string;
+  backupId?: number;
+  backuptime?: number;
+  path?: string | null;
+}) {
+  const backupsUrl = `${BASE_HREF}/backups`;
+  const clientUrl = `${backupsUrl}/${clientId}`;
+
+  const baseBreadcrumbItems: BreadcrumbItem[] = [
+    {
+      key: crypto.randomUUID(),
```

Comment:
I'm guessing you could use a hard-coded string and in the second case a prefixed client name as key?

Reply:
Yes, good point, I updated the base breadcrumb items (non-paths) to use strings.